Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs

```diff
@@ -81,7 +81,13 @@ protected override void OnStart()
                 {
                     Distribute.DisableAutomaticCheckForUpdate();
                 }
+                Task<bool> storageTask = null;
+                if (Current.Properties.ContainsKey(Constants.StorageMaxSize) && Current.Properties[Constants.StorageMaxSize] is long size)
+                {
+                    storageTask = AppCenter.SetMaxStorageSizeAsync(size);
+                }
                 AppCenter.Start(GetTokensString(), typeof(Analytics), typeof(Crashes), typeof(Distribute));
+                storageTask?.Wait();
```

Comment:
Why do you need it here? I guess fire and forget works fine

Reply:
Fixed